### DISCUSSION TITLE
Welcome to tabler Discussions!

### DISCUSSION THREAD

**Discussion by codecalm** [tabler/tabler#735] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TrexKikBut** [comment]
Hello!

I'm wanting to use this dashboard on my ASP.NET project. We use an older version of bootstrap, and I want to update it all to this layout. When I add the folders to my project, like src, dist, etc. then try to use index.html, all of the scripts and directories are pointing at a different location.

How can I update all of the locations so that they know where to point, because my file setup is different. 

I have this type of setup:
- Content/dist
- Content/src
- index.html

Thank you for the help and I love this layout!

**Comment by nejcdev** [comment]
Hello do you have discord channel for this project ?
